sample.env

```diff
@@ -58,3 +58,17 @@ POSTGRES_VERSION=14
 ##
 BITCOIN_BLOCKCHAIN_FOLDER=./persistent-data/blockchain-bitcoin
 BITCOIN_CONFIG_FILE=../conf/${NETWORK}/bitcoin.conf
+
+## Mainnet Defaults
+BTC_HOST=bitcoin.mainnet.stacks.org
+BTC_RPC_USER=stacks-mainnet
```

Comment:
I'd keep this simple - the rpc user/pass should be kept simple by default (and can be re-used for default setup). 
i'd argue using the existing `stacks:foundation` rpc auth would be fine

Reply:
I agree. That's how I had it initially! Changed it back.